solutions/src/main/java/ru/sberbank/school/task07/WordsFrequencyImpl.java

```diff
@@ -0,0 +1,31 @@
+package ru.sberbank.school.task07;
+
+import lombok.NonNull;
+import ru.sberbank.school.task07.utill.LineParser;
+
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class WordsFrequencyImpl implements WordFrequency {
+    private FileParser parser;
+
+    public WordsFrequencyImpl(@NonNull FileParser fileParser) {
+        this.parser = fileParser;
+    }
+
+    @Override
+    public Map<String,Integer> countWords(String pathToFile) throws FileNotFoundException {
+        List<String> list = parser.parse(pathToFile);
+
+        Map<String,Integer> map = new HashMap<>();
+        for (String l : list) {
+            for (String w : LineParser.getWords(l.toLowerCase())) {
+                map.put(w, map.getOrDefault(w, 0)+1);
```

Comment:
WhitespaceAround: &#39;+&#39; is not followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1998554474>*